extensions/amp-story/amp-story-analytics.md

```diff
@@ -178,6 +153,29 @@ The `story-page-attachment-enter` trigger is fired when a page attachment is ope
 
 The `story-page-attachment-exit` trigger is fired when a page attachment is dismissed by the user.
 
+### Story shopping product listing page view trigger (`"on": "story-shopping-plp-view"`)
+
+Triggers whenever amp-story-shopping-attachment displays the Product Listing Page without an active product.
```

Comment:
Please follow the writing style of the document.

"The `<event name>` trigger is fired when..."